core/src/main/python/create.py

```diff
@@ -376,4 +376,7 @@ def main(args):
 
 if __name__ == '__main__' or __name__ == 'main':
     WebLogicDeployToolingVersion.logVersionInfo(_program_name)
-    main(sys.argv)
+    try:
+        main(sys.argv)
+    except:
+        exception_helper.__handleUnexpectedException(sys.exc_info(), _program_name, _class_name, __logger)
```

Comment:
ok, but what do you catch?  except Exception doesn't necessarily catch everything. There are mixed opinions about what to catch in this case.  Do you catch Exception, which doesn't get all unhandled exceptions, or BaseException?  One advantage of Exception I suppose is that you don't get system exit conditions, which may (or may not) address the issue Richard points out.  The disadvantage of doing it as is, is that you can catch keyboard interrupts.  I'm open to suggestions, having frankly, not dived into this deeply before this.

Reply:
I've simplified it.

Reply:
To catch all java exceptions we may need to add java.lang.Throwable into the except clause in addition to Exception.  It depends on whether we think Exception is good enough to catch most unexpected exceptions or not.

Reply:
I think catching java.lang.Exception should be sufficient since I believe all Jython exceptions derive from it.  Yes, technically an Error could make it through but I don’t think that is our main issue.